tests/requirements/base.txt

```diff
@@ -1,9 +1,11 @@
-django-select2>=5.11,<7 # 7 works in Django 2+
 django-filer>=1.5.0
 html5lib>=1
-django-app-helper
 tox
 coverage
 isort
 flake8
+flake8-pyproject
```

Comment:
I've seen this in the pre-commit hooks, but is it used outside of those as well to be needed here?

Reply:
Yes, it's a plugin that make flake8 read the settings from pyproject.toml, where its settings are kept.